modules/tinymce/changelog.txt

```diff
@@ -5,6 +5,8 @@ Version 5.5.0 (TBD)
     Added the ability to remove images on a failed upload using the `images_upload_handler` failure callback #TINY-6011
     Added `hasPlugin` function to the editor API to determine if a plugin exists or not #TINY-766
     Added new `ToggleToolbarDrawer` command and query state handler to allow the toolbar drawer to be programmatically toggled and its toggle state to be checked #TINY-6032
+    Added the ability to use `colgroup` elements in tables #TINY-6050
+    Added a new setting `table_use_colgroups` for toggling the behavior #TINY-6050
```

Comment:
```suggestion
    Added a new setting `table_use_colgroups` for toggling whether colgroups are used in new tables #TINY-6050
```

Reply:
@HAFRMO @lnewson @metricjs

Reply:
I do wonder a bit if we should mention that it's about column width things, but that might make it rather long and complicated. I'm okay with this.